pkgs/tools/package-management/nix-visualize/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, nix
+, python
+, matplotlib
+, networkx
+, pandas
+, pygraphviz
+}:
+
+python.pkgs.buildPythonApplication rec {
+  version = "1.0.5-unstable-2024-01-17";
+  pname = "nix-visualize";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "craigmbooth";
+    repo = "nix-visualize";
+    rev = "5b9beae330ac940df56433d347494505e2038904";
+    hash = "sha256-VgEsR/Odddc7v6oq2tNcVwCYm08PhiqhZJueuEYCR0o=";
+  };
+
+  postInstall = ''
+    wrapProgram $out/bin/nix-visualize \
+        --prefix PATH : ${lib.makeBinPath [nix]}
```

Comment:
We use two spaces in shell to intend. Also do we really need to prefix nix here? It is almost necessary that people already have that in PATH.

Reply:
I wanted to include `nix` here to explicitly state the dependency. It's true it should already be in the PATH, and we can leave it out if you prefer not having it explicitly stated the way it now is.